<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsStorageControllers</class>
 <widget class="QWidget" name="SettingsStorageControllers">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>496</width>
    <height>449</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="labelFD">
       <property name="text">
        <string>FD Controller:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="comboBoxFD">
       <property name="maxVisibleItems">
        <number>30</number>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="pushButtonFD">
       <property name="text">
        <string>Configure</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labelCDInterface">
       <property name="text">
        <string>CD-ROM Controller:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="comboBoxCDInterface">
       <property name="maxVisibleItems">
        <number>30</number>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="pushButtonCDInterface">
       <property name="text">
        <string>Configure</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBoxHD">
     <property name="title">
      <string>Hard disk</string>
     </property>
     <layout class="QGridLayout" name="gridLayoutHD">
      <item row="0" column="0">
       <widget class="QLabel" name="labelHD1">
        <property name="text">
         <string>Controller 1:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QComboBox" name="comboBoxHD1">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maxVisibleItems">
         <number>30</number>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QPushButton" name="pushButtonHD1">
        <property name="text">
         <string>Configure</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelHD2">
        <property name="text">
         <string>Controller 2:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QComboBox" name="comboBoxHD2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maxVisibleItems">
         <number>30</number>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QPushButton" name="pushButtonHD2">
        <property name="text">
         <string>Configure</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="labelHD3">
        <property name="text">
         <string>Controller 3:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QComboBox" name="comboBoxHD3">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maxVisibleItems">
         <number>30</number>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QPushButton" name="pushButtonHD3">
        <property name="text">
         <string>Configure</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="labelHD4">
        <property name="text">
         <string>Controller 4:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QComboBox" name="comboBoxHD4">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maxVisibleItems">
         <number>30</number>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QPushButton" name="pushButtonHD4">
        <property name="text">
         <string>Configure</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBoxSCSI">
     <property name="title">
      <string>SCSI</string>
     </property>
     <layout class="QGridLayout" name="gridLayoutSCSI">
      <item row="0" column="0">
       <widget class="QLabel" name="labelSCSI1">
        <property name="text">
         <string>Controller 1:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QComboBox" name="comboBoxSCSI1">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maxVisibleItems">
         <number>30</number>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QPushButton" name="pushButtonSCSI1">
        <property name="text">
         <string>Configure</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelSCSI2">
        <property name="text">
         <string>Controller 2:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QComboBox" name="comboBoxSCSI2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maxVisibleItems">
         <number>30</number>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QPushButton" name="pushButtonSCSI2">
        <property name="text">
         <string>Configure</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="labelSCSI3">
        <property name="text">
         <string>Controller 3:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QComboBox" name="comboBoxSCSI3">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maxVisibleItems">
         <number>30</number>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QPushButton" name="pushButtonSCSI3">
        <property name="text">
         <string>Configure</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="labelSCSI4">
        <property name="text">
         <string>Controller 4:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QComboBox" name="comboBoxSCSI4">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maxVisibleItems">
         <number>30</number>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QPushButton" name="pushButtonSCSI4">
        <property name="text">
         <string>Configure</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBoxCassette">
     <property name="text">
      <string>Cassette</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>comboBoxFD</tabstop>
  <tabstop>pushButtonFD</tabstop>
  <tabstop>comboBoxCDInterface</tabstop>
  <tabstop>pushButtonCDInterface</tabstop>
  <tabstop>comboBoxHD1</tabstop>
  <tabstop>pushButtonHD1</tabstop>
  <tabstop>comboBoxHD2</tabstop>
  <tabstop>pushButtonHD2</tabstop>
  <tabstop>comboBoxHD3</tabstop>
  <tabstop>pushButtonHD3</tabstop>
  <tabstop>comboBoxHD4</tabstop>
  <tabstop>pushButtonHD4</tabstop>
  <tabstop>comboBoxSCSI1</tabstop>
  <tabstop>pushButtonSCSI1</tabstop>
  <tabstop>comboBoxSCSI2</tabstop>
  <tabstop>pushButtonSCSI2</tabstop>
  <tabstop>comboBoxSCSI3</tabstop>
  <tabstop>pushButtonSCSI3</tabstop>
  <tabstop>comboBoxSCSI4</tabstop>
  <tabstop>pushButtonSCSI4</tabstop>
  <tabstop>checkBoxCassette</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
